doc/bucket-class-crd.md

```diff
@@ -8,6 +8,12 @@ Data placement capabilities are built as a multi-layer structure, here are the l
 - Mirroring Layer - list of spread-layers, async-mirroring to all mirrors, with locality optimization.
 - Tiering Layer - list of mirroring-layers, push cold data to next tier.
 
+Constraints:
+You can use the same backing-store in multiple bucket-classes.
+The same backing-store cannot be used more than once within a bucket-class configuration.
+Currently the operator cli only supports single tier spread/mirror configurations.
+In order to configure multiple tiers you will need to prepare a bucket-class YAML and create it.
+
```

Comment:
Minor suggestion:

Constraints:
A backing-store name may appear in more than one bucket-class but may not appear more than once
in a single bucket-class.
The operator cli currently only supports a single tier placement-policy for a bucket-class. YAML must
be used to create a bucket-class with a placement-policy that has multiple tiers.